draft-ietf-rtcweb-jsep.xml

```diff
@@ -3186,11 +3186,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <list>
 
             <t>The "c=" line MUST be checked for syntax but its value
-            is not used. This supersedes the guidance in
-            <xref target="RFC5245" />, Section 6.1, to use
-            "ice-mismatch" to indicate mismatches between "c=" and the
-            candidate lines; because JSEP always uses ICE,
-            "ice-mismatch" is not useful in this context.</t>
+            is only used for ICE mismatch detection. Some
+            implementations do not bother to implement ICE mismatch
```

Comment:
I would rewrite the second sentence as:

"Note that JSEP implementations should never encounter this condition because ICE is required for WebRTC."